assume/common/outputs.py

```diff
@@ -40,18 +40,18 @@ def __init__(
         start: datetime,
         end: datetime,
         db_engine=None,
-        export_csv_path: str = "",
-        save_frequency_hours: int = 24,
+        export_csv_path: str = None,
+        save_frequency_hours: int = None,
```

Comment:
export_csv_path = "" did the job, why the change..?

Reply:
for me it always creates an empty folder in the root without a name and writes the files over there. So it doesn't work

Reply:
I don't think that we should set this to None..